spec/relativetimeformat.html

```diff
@@ -1,61 +1,261 @@
+            1. If Type(_subst_) is List, then 
+              1. Let _len_ be ? ToLength(? Get(_subst_, *"length"*)).
+              1. Set _i_ to *0*.
+              1. Repeat, while _i_ &lt; *0*
+                1. Add _subst_.[_i_] as a new element on the list _result_.
+            1. Else,
+              1. Add _subst_ as a new element on the list _result_.
+        1. Return _result_.
+      </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-FormatRelativeTime" aoid="FormatRelativeTime">
+      <h1>FormatRelativeTime (relativeTimeFormat, x)</h1>
+
+      <p>
+        The FormatRelativeTime abstract operation is called with arguments _relativeTimeFormat_ (which must be an object initialized as a RelativeTimeFormat) and _x_ (which must be a Number value), and performs the following steps:
+      </p>
+
+      <emu-alg>
+        1. Let _parts_ be ? PartitionRelativeTimePattern(_relativeTimeFormat_, _x_).
+        1. Let _result_ be an empty String.
+        1. For each _part_ in _parts_, do:
+          1. Set _result_ to a String value produced by concatenating _result_ and _part_.[[Value]].
+        1. Return _result_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-FormatRelativeTimeToParts" aoid="FormatRelativeTimeToParts">
+      <h1>FormatRelativeTimeToParts (relativeTimeFormat, x)</h1>
+        The FormatRelativeTimeToParts abstract operation is called with arguments _relativeTimeFormat_ (which must be an object initialized as a RelativeTimeFormat) and _x_ (which must be a Number value), and performs the following steps:
+      <p>
+      </p>
+
+      <emu-alg>
+        1. Let _parts_ be ? PartitionRelativeTimePattern(_relativeTimeFormat_, _x_).
+        1. Let _result_ be ArrayCreate(0).
+        1. Let _n_ be 0.
+        1. For each _part_ in _parts_, do:
+          1. Let _O_ be ObjectCreate(%ObjectPrototype%).
+          1. Perform ? CreateDataPropertyOrThrow(_O_, "type", _part_.[[Type]]).
+          1. Perform ? CreateDataPropertyOrThrow(_O_, "value", _part_.[[Value]]).
+          1. Perform ? CreateDataProperty(_result_, ? ToString(_n_), _O_).
+          1. Increment _n_ by 1.
+        1. Return _result_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-GetBestMatchUnit" aoid="GetBestMatchUnit">
+      <h1>GetBestMatchUnit (units)</h1>
+
+      <p>
+        When the GetBestMatchUnit abstract operation is called with argument _units_, the following steps are taken:
+      </p>
+
+      <emu-alg>
+        1. Assert: Type(_units_) is a Record.
+        1. If _units_.[[Second]] is less than 60, return "Second".
+        1. If _units_.[[Minute]] is less than 60, return "Minute".
+        1. If _units_.[[Hour]] is less than 24, return "Hour".
+        1. If _units_.[[Day]] is less than 7, return "Day".
+        1. If _units_.[[Week]] is less than 4, return "Week".
+        1. If _units_.[[Month]] is less than 12, return "Month".
+        1. If _units_.[[Quarter]] is less than 4, return "Quarter".
```

Comment:
Is it my impression or `Quarter` is never reached given `12 months == 4 quarters`.